.github/workflows/ci.yml

```
@@ -1,210 +1,230 @@
-
-      - name: Run SDK tests
-        working-directory: sdk
-        run: bun run test
-
-      - name: Run CLI tests
-        working-directory: cli
+          echo "🚀 Starting Solana test validator..."
+          solana-test-validator --reset --quiet --log /tmp/validator.log &
+          VALIDATOR_PID=$!
+          echo "Validator PID: $VALIDATOR_PID"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)